### ISSUE TITLE
VB rules missing from .editorConfig

### ISSUE THREAD

**Issue by paul1956** [dotnet/winforms#2712]
**Is your feature request related to a problem? Please describe.**
VB code does not have .editorConfig entries causing warnings/suggestions and poor code consistency when opened in VS.

**Describe the solution you'd like**
Add below above first dotnet_ entry
```
# C# and Visual Basic files
[*.{cs,vb}]
charset = utf-8-bom
```
Move any csharp_style entries to [*.cs] section
Add below
```
# Visual Basic files
[*.vb]
# Modifier preferences
visual_basic_preferred_modifier_order = Partial,Default,Private,Protected,Public,Friend,NotOverridable,Overridable,MustOverride,Overloads,Overrides,MustInherit,NotInheritable,Static,Shared,Shadows,ReadOnly,WriteOnly,Dim,Const,WithEvents,Widening,Narrowing,Custom,Async:suggestion
```

**Describe alternatives you've considered**
Use Dotnet/Samples .editorConfig file or make sure both are the same
**Will this feature affect UI controls?**
No

**Comment by RussKie** [comment]
@paul1956 are you referring to [our .editorconfig](https://github.com/dotnet/winforms/blob/master/.editorconfig)?
You seem to have a good idea of what's required, why not send a PR in?

**Comment by paul1956** [comment]
@RussKie I just did a PR, there are some issues in the old and new file that I have commented on, in the document, but an not qualified to fix. I expect one more commit to fix and remove the comment.

**Comment by paul1956** [comment]
Closed via PR #2715
